boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
+#define PWM_NUMOF           (1U)
+#define PWM_0_EN            (1)
+#define PWM_MAX_VALUE       (0xffff)
+//#define PWM_MAX_CHANNELS    4
```

Comment:
That's part of the WIP. Not using it now, but will maybe later.
Also unshure about the number.
The pwm has 8 channels. For the actual implementation I relied on [Pin mapping in RIOT](https://docs.google.com/spreadsheets/d/1g5xMpLbi6cMc9QdHQrpf3XH_hiRb4wrAu1wvvpelMUw/edit?pli=1#gid=1892583546) and used channels 4-7 for output.